src/config/modmap_action.rs

```diff
@@ -19,10 +19,8 @@ pub enum ModmapAction {
 #[serde_as]
 #[derive(Clone, Debug, Deserialize)]
 pub struct MultiPurposeKey {
-    #[serde(deserialize_with = "deserialize_key")]
-    pub held: Key,
-    #[serde(deserialize_with = "deserialize_key")]
-    pub alone: Key,
+    pub held: Keys,
```

Comment:
Could you also accept a non-vector `Key` for backward compatibility? For reference, we do something similar in multiple places, e.g. `Application::only`. They accept a string but also a list of strings in the same place. Many people could still use this feature without specifying multiple keys here, so that would be useful.

Reply:
Keys is an emun, it can be a non-vector 'Key'
```
pub enum Keys {
    #[serde(deserialize_with = "deserialize_key")]
    Key(Key),
    #[serde(deserialize_with = "deserialize_virtual_modifiers")]
    Keys(Vec<Key>),
}
```

Reply:
Oh okay, sorry about that. I got confused by changes in the documentation and tests.